tests/Resources/ti.ui.android.progressindicator.addontest.js

```diff
@@ -0,0 +1,211 @@
+/*
+ * Appcelerator Titanium Mobile
+ * Copyright (c) 2019 by Axway, Inc. All Rights Reserved.
+ * Licensed under the terms of the Apache Public License
+ * Please see the LICENSE included with this distribution for details.
+ */
+/* eslint-env mocha */
+/* global Titanium */
```

Comment:
<!--
  0 failure: 
  1 warning:  tests/Resources/t...
  
  
  DangerID: danger-id-Danger;
  File: tests/Resources/ti.ui.android.progressindicator.addontest.js;
  Line: 8;
-->

- :warning: tests/Resources/ti.ui.android.progressindicator.addontest.js line 8 – 'Titanium' is defined but never used. (no-unused-vars)